### PR TITLE
added minified webpack configuration & distribution

### PR DESCRIPTION
**Pull request by vincentriemer**

### CODE REVIEW

**Comment by alexlande** [comment]
Thanks for this! Could you make two changes before I merge it in?

- [ ] Revert the changes to the two dist files (want to only build them when we tag a version to reduce repo churn)
- [ ] Update the minified webpack config to extend the base one with `_.assign()` to DRY things out a little bit.

**Comment by vincentriemer** [comment]
Changes made, only thing different from what you suggested was using _.merge() instead of _.assign()

**Comment by alexlande** [comment]
Perfect! Thanks a lot.

File: webpack.config.minified.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+var webpack = require('webpack');
+var _ = require('lodash');
+
+module.exports = _.merge(require('./webpack.config.js'), {
```
Comment (ryan-roemer) [inline webpack.config.minified.js]: Switch to

```js
module.exports = _.merge({}, require('./webpack.config.js'), {
```

If you end up using 2+ configurations in the same process (aka, what often happens in gulp-land...)

@alexlande -- Want to ticket that out?

Comment (alexlande) [inline webpack.config.minified.js]: Ah, good call. Will do.

File: webpack.config.minified.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  output: {
+    filename: 'radium.min.js'
+  },
+  plugins: [
+    new webpack.optimize.UglifyJsPlugin({
```
Comment (ryan-roemer) [inline webpack.config.minified.js]: Also suggest:

```js
new webpack.optimize.DedupePlugin()
```

now and in the future, consider writing "dev-only" code against a `production.env.NODE_ENV` check and then defining this:

```js
    new webpack.DefinePlugin({
      "process.env": {
        // Signal production mode for React JS and other libs.
        NODE_ENV: JSON.stringify("production")
    })
```

As that's more-or-less the react standard now...

**Pull request merged**